### PR TITLE
CDAP-8692 Cask Market v2 changes

### PR DESCRIPTION
**Pull request by awholegunch**
Adds a version to the Cask Market to the vars
Adds support for a local build to the integrations manual
Adds new actions and revises current in Cask Market

Building as a Quick Build: http://builds.cask.co/browse/CDAP-DQB282-1 (passed)

Page of interest: _to be completed_

Fix for: https://issues.cask.co/browse/CDAP-8692

### CODE REVIEW

File: cdap-docs/integrations/source/cask-market.rst [hunk 1]
```diff
@@ -593,6 +671,42 @@ Example action:
...
+.. highlight:: json-ellipsis
+
+Example action:
+
+.. container:: highlight
+
+  .. parsed-literal::
+
+    {
+      "type": "deploy_app",
```
Comment (albertshau) [inline cdap-docs/integrations/source/cask-market.rst]: one_step_deploy_app

Comment (awholegunch) [reply cdap-docs/integrations/source/cask-market.rst]: Fixed.

File: cdap-docs/integrations/source/cask-market.rst [hunk 2]
```diff
@@ -184,7 +190,7 @@ and a list of actions required to install the package:
             {
               "name": "steps",
               "value": [
-                "Download the ZIP file from MySQL at https://dev.mysql.com/downloads/file/?id=462850",
+                "Download the ZIP file from MySQL at \https://dev.mysql.com/downloads/file/?id=462850",
```
Comment (albertshau) [inline cdap-docs/integrations/source/cask-market.rst]: how come this backslash is needed?

**Review by albertshau — APPROVED** [review]
one question, otherwise lgtm

File: cdap-docs/integrations/source/cask-market.rst (hunk 2 above)
Comment (awholegunch) [reply cdap-docs/integrations/source/cask-market.rst]: If there is no backslash, Sphinx turns this into an active HTML link. That might be OK, but I have tried to not have active links in the code samples. There are arguments pro and con: pro is that a link checker can see it and actually test if it works. Cons is that it looks distracting. It's actually not an active link in the UI, which is weird IMO.

Comment (awholegunch) [reply cdap-docs/integrations/source/cask-market.rst]: I've decided to remove the backslash, so the link is active and can be checked.

**Pull request merged**